fhir-mapping/src/test/ml-modules/root/test/suites/testLogger.sjs

```diff
@@ -0,0 +1,276 @@
+const test = require('/test/test-helper.xqy');
```

Comment:
I haven't fully digested the logger, but am hesitant to add a custom logger vs the normal xdmp logging and tracing. We should get together and all review and think about logging long-term.